synapse/cores/common.py

```diff
@@ -900,7 +900,7 @@ def setRowsByIdProp(self, iden, prop, valu):
         self.savebus.fire('core:save:set:rows:by:idprop', iden=iden, prop=prop, valu=valu)
         self._setRowsByIdProp(iden, prop, valu)
 
-    def getRowsByProp(self, prop, valu=None, mintime=None, maxtime=None, limit=None):
+    def getRowsByProp(self, prop, valu=None, mintime=None, maxtime=None, timeout=None, limit=None):
```

Comment:
For API stability, new kwargs options should be added at the end if possible to account for API users which used assumed ordering.